loading.bs

```diff
@@ -331,6 +392,417 @@ Note: Applying the signed exchange's response here has the effect of letting a
+        1. For each |link| of |links|:
+            1. If |link|'s [=Relation Type=] is not 'preload', then continue.
+            1. Let |linkTarget| be |link|'s [=Link Target=].
+            1. Let |asAttribute| be |link|'s [=Target Attribute named=] `"as"`.
+            1. If |asAttribute| is null, then continue.
+            1. If |asAttribute| is `"image"`, then:
+                1. Let |imagesrcset| be |link|'s [=Target Attribute named=]
+                    `"imagesrcset"`.
+                1. Let |imagesizes| be |link|'s [=Target Attribute named=]
+                    `"imagesizes"`.
+                1. Create a <{link}> element |linkElement| whose <{link/href}>
+                    attribute is |linkTarget|, and <{link/imagesrcset}>
+                    attribute is |imagesrcset|, and <{link/imagesizes}>
+                    attribute is |imagesizes|.
+                1. Let |selected source| and <var ignore=''>selected pixel
+                    density</var> be the URL and pixel density that results from
+                    [=selecting an image source=] given |linkElement|,
+                    respectively.
+                1. If |selected source| is not null, then set |linkTarget| to
+                    the result of [=URL parser|parsing=] |selected source|, with
+                    a base URL of |navigationParams|'s [=navigation
+                    params/response=]'s [=response/URL=].
+            1. Let |headerIntegrity| be null.
+            1. For each |allowedSxgLink| of |allowedSxgLinks|:
+                1. Let |storedExchange| be the result of creating an
+                    [=exchange=] with the URL of |allowedSxgLink|'s [=allowed
+                    signed exchange link info/target=] and a new [=response=].
+                1. If |allowedSxgLink|'s [=allowed signed exchange link
+                    info/variants=] is null, [=header list/set=] `` `Variants`
+                    ``/|allowedSxgLink|'s [=allowed signed exchange link
+                    info/variants=] in |storedExchange|'s
+                    [=exchange/response=]'s [=response/header list=].
+                1. If |allowedSxgLink|'s [=allowed signed exchange link
+                    info/variant key=] is not null, [=header list/set=] ``
+                    `Variant-Key` ``/|allowedSxgLink|'s [=allowed signed
+                    exchange link info/variant key=] in |storedExchange|'s
+                    [=exchange/response=]'s [=response/header list=].
+                1. Let |requestForMatch| be the result of [=creating a
+                    potential-CORS request=] given |allowedSxgLink|'s [=allowed
+                    signed exchange link info/target=], the empty string, and
+                    [=No CORS=].
+                1. If |requestForMatch| [=doesn't match the stored exchange=]
+                    |storedExchange|, then continue.
+                1. Set |headerIntegrity| to |allowedSxgLink|'s [=allowed signed
```

Comment:
This looks like it'll use the `header-integrity` of the last matching `allowed-alt-sxg` link if multiple such links match. Does that match your implementation?

Reply:
No. Chrome is using the first allowed-alt-sxg link entry.h.

I will update the spec to make it closer to the Chrome's implementation.
(But it may take a few months because of my paternity leave.)